pkg/reconciler/filter.go

```diff
@@ -34,3 +35,12 @@ func AnnotationFilterFunc(key string, value string, allowUnset bool) func(interf
 		return false
 	}
 }
+
+// LabelExistsFilterFunc creates a FilterFunc only accepting objects which have a given label.
+func LabelExistsFilterFunc(label string) func(obj interface{}) bool {
+	return func(obj interface{}) bool {
+		endpoints := obj.(*corev1.Endpoints)
```

Comment:
This is not generalized. This function can operate on any metav1.Object (see the function above), like

```golang
func LabelExistsFilterFunc(label string) func(obj interface{}) bool {
	return func(obj interface{}) bool {
		if mo, ok := obj.(metav1.Object); ok {
			labels := mo.GetLabels()
			_, ok := labels[label]
			return ok
		}
		return false
	}
}
```